src/templates/layouts/event/index.test.tsx

```diff
@@ -212,4 +212,34 @@ describe('<Event /> Component', () => {
     render(<Event {...data} />)
     expect(screen.getByText('3307 10th Avenue Southeast')).toBeInTheDocument()
   })
+
+  describe('va-links in paragraph tags', () => {
+    it('always wraps outreach link in paragraph tag', () => {
+      render(<Event {...data} />)
+      const outreachLink = document.querySelector(
+        'va-link[href="/outreach-and-events/events/"]'
+      )
```

Comment:
While it's totally fine in this case to use `document.querySelector()`, `container.querySelector()` is maybe a "better" way.

Under the hood, `render()` [technically _appends_ the elements](https://testing-library.com/docs/react-testing-library/api#render) to the end of `document.body`, so if there's anything in there already for whatever reason, `document.querySelector()` will pick it up, but `container.querySelector()` won't since [`container` is only the DOM node](https://testing-library.com/docs/react-testing-library/api#container-1) we're trying to render for this test.